client/js/components/procedure/StatementSegmentsList/SegmentLocationMap.vue

```diff
@@ -249,13 +265,24 @@ export default {
           polygon: JSON.stringify(this.featuresObject)
         }
       })
+      const comments = { ...this.segment.relationships.comments } || null
+
+      /**
+       *  Comments need to be removed as updating them is technically not supported
+       *  After completing the request, they are added again to the store to be able to display them
+       */
+      if (this.segment.relationships.comments) {
+        delete this.segment.relationships.comments
+      }
 
       return this.saveSegmentAction(this.segmentId)
         .then(checkResponse)
         .then(() => {
+          this.restoreComments(comments)
           dplan.notify.confirm(Translator.trans('confirm.saved'))
         })
         .catch(() => {
+          this.restoreComments(comments)
```

Comment:
You can also use `.finally()` to avoid the repetition.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/finally

Reply:
Thanks! Done in [8f9c738](https://github.com/demos-europe/demosplan-core/pull/3468/commits/8f9c73861bb42af6f4aab9169d7a3155a5c52316)